src/view/home.js

```diff
@@ -1,87 +1,105 @@
 (function(){
-  LifeGamification.home = {};
+  const welcome_message = `
+    <p>Welcome to Life Gamification!<br>
+    To add your first skill go to Edit skills.</p>`;
 
-  //Functions to both Home and Edit views.
+  class SkillView extends Abstract.View {
+    _addTitle() {
+      this._appendNewElement(
+        'a', {"class": "skill__level-number"}, this.model.level);
+      this._appendNewElement(
+        'a', {"class": "skill__level-text"}, "lvl");
+      this._appendNewElement('a', {"class": "skill__name"}, this.model.name);
+    }
 
-  LifeGamification.home.viewLevelAndExp = function (skill) {
-    const number = LifeGamification.skillsView.findIndex(function (element) {
-      return element === skill;
-    });
-    $(`.level${number}`).html(`${skill.level}`);
-    $(`.name${number}`).html(`${skill.name}`);
-    $(`.exp${number}`).html(`
-      ${skill.expInThisLevel}/${skill.expTillNextLevel}`);
+    _addExpTillNextLevel() {
+      this._appendNewElement(
+        'a', {"class": "skill_experience"},
+        `${this.model.expInThisLevel}/${this.model.expTillNextLevel}`,
+      );
+    }
 
-    let percent = Math.floor(
-      100 * skill.expInThisLevel / skill.expTillNextLevel);
-    $(`.fill${number}`).html(`${percent}%`);
-    $(`.fill${number}`).css('width', `${percent}%`);
-  }
+    render() {
+      this._addTitle();
+      this._addProgressBar();
+      this._addExpTillNextLevel();
+    }
 
-  LifeGamification.home.skillHTML = function (number) {
-    return (`
-      <a class="skill__level-number level${number}">-1</a>
-      <a class="skill__level-text">lvl</a>
-      <a class="skill__name name${number}">Skillname</a>
-      <div class="progress-bar__wrapper">
-          <span class="progress-bar__container">
-              <span class="progress-bar__container-fill fill${number}">-1%</span>
-          </span>
-          <div class="progress-bar__buttons">
-              <input class="progress-bar__add-input" id="addVal${number}" type="number" value="1">
-              <span class="progress-bar__add-button" id="add${number}"> +</span>
-          </div>
-        </div>
-        <a class="skill__experience exp${number}">-1/-1</a>
-    </div>
-    `);
-  }
+    _addProgressBar() {
+      const barWrapper = this._appendNewElement(
+        'div', {"class": "progress-bar__wrapper"}, '');
+      const container = this._createElement(
+        'span', {"class": "progress-bar__container"}, '', barWrapper);
```

Comment:
There exists a html tag `progress`. I don't know how easy it is to style it but we should consider using this, instead of making our own progress with spans and div's.

It may be not compatible with Jurek's design.